ui/src/app/components/apps-panel.tsx

```diff
@@ -63,6 +68,8 @@ export const AppsPanel = (props: {
   const { onClose, workspace } = props;
   const { runtime } = useStore(runtimeStore);
   const { config } = useStore(serverConfigStore);
+  // all GKE apps (not Jupyter)
+  const { userApps } = useStore(userAppsStore);
```

Comment:
Do we have an ontology here? For example, Set(allUserApps) = Set(gkeUserApps) + Set(nonGkeUserApps)?

Reply:
the terms "user apps" and "GKE apps" are equivalent.  Jupyter uses (GCE or DataProc) "runtimes" instead of (GKE) "user apps"

User Environments is the set of both (user apps + runtimes)